build.gradle.kts

```diff
@@ -11,23 +10,28 @@ val githubPassword: String = project.findProperty("gpr.key") as? String ?: Syste
 
 repositories {
     mavenCentral()
-    maven {
-        url = uri("https://maven.pkg.github.com/revanced/multidexlib2")
+    fun githubPackages(repo: String) = maven {
+        url = uri("https://maven.pkg.github.com/revanced/$repo")
         credentials {
             username = githubUsername
             password = githubPassword
         }
     }
+
+    githubPackages("multidexlib2")
+    githubPackages("ARSCLib")
 }
 
 dependencies {
     implementation("xpp3:xpp3:1.1.4c")
     implementation("app.revanced:smali:2.5.3-a3836654")
     implementation("app.revanced:multidexlib2:2.5.3-a3836654")
-    implementation("app.revanced:apktool-lib:2.7.0")
+    implementation("app.revanced:arsclib:1.1.6")
 
     implementation("org.jetbrains.kotlin:kotlin-reflect:1.8.10")
     testImplementation("org.jetbrains.kotlin:kotlin-test:1.8.10")
+
+    compileOnly("com.google.android:android:4.1.1.4")
```

Comment:
It appears to be used by the `PatchBundle` class to gain access to `DexClassLoader`

Reply:
Why was this not an issue before and only just now?

Reply:
> Why was this not an issue before and only just now?

Because the patcher user supplied the `ClassLoader` instead, which means we did not need to access any dalvik class loaders